### PR TITLE
Minor tweaks for login modals

### PR DESCRIPTION
**Pull request by jstnjns**
## Overview
Updates to the component library realized when implementing for login modals

### CODE REVIEW

File: src/components/InputField/index.js [hunk 1]
```diff
@@ -12,10 +12,14 @@ const FIELD_PROP_TYPE = {
 
 const renderInput = ({
   input,
+  meta: {
```
Comment (missx) [inline src/components/InputField/index.js]: Can yo add this as a prop type also?

File: src/components/Input/index.js [hunk 2]
```diff
@@ -113,12 +113,12 @@ export default class Input extends PureComponent {
...
             />
 
-            {label &&
+            {(error || label) &&
               <label
                 htmlFor={name}
                 className='mc-form-input__label'
               >
-                {label}
+                {label} {error}
```
Comment (bernabe9) [inline src/components/Input/index.js]:
```suggestion
                {error || label}
```

File: src/components/InputField/index.js (hunk 1 above)
Comment (jstnjns) [reply src/components/InputField/index.js]: yes, good call

File: src/components/Input/index.js (hunk 2 above)
Comment (jstnjns) [reply src/components/Input/index.js]: I, initially, had exactly that.  Errors that come from the server are actually incredibly useful in the format they are in:

```json
{ "email": "can't be blank" }
```

leads well to be rendered like `Email can't be blank`

Comment (jstnjns) [reply src/components/Input/index.js]: My idea fails if the label isn't the name of the field.

`Tell us how you would describe yourself can't be blank` fails...  Changing.  Thanks!

**Pull request merged**